docs/intro.rst

```diff
@@ -372,6 +372,8 @@ This is the current list of error and warning codes:
 +----------+----------------------------------------------------------------------+
 | E902     | IOError                                                              |
 +----------+----------------------------------------------------------------------+
+| E999     | Triple quoted string uses simple quotes                              |
```

Comment:
I decided to chose a number I'd be able to recognise. Any suggestion is welcome. It doesn't seem to fit in any existing category so I'll leave you guys to decide.